src/components/form/select/select.js

```diff
@@ -0,0 +1,161 @@
+import React from 'react';
+import styled, { css } from 'styled-components';
+
+const Select = ({ variant, title, optional, options, ...props }) => {
+    
+    const [selected, setSelected] = React.useState({ id : null, image : '/assets/icons/n_sample_img.svg', value : 'select' });
+    const [open, setOpen] = React.useState(false);
+
+    const toggleOpen = () => {
+        setOpen(!open);
+    }
+
+    const selectOption = (option) => {
+        setSelected(option);
+        toggleOpen();
+    }
+
+    return (
+        <Container>
+            <StyledLabel>{ title } <span hidden={ !optional }>(optional)</span> </StyledLabel>
+            <StyledSelect variant={ variant } { ...props } onClick={ toggleOpen }>
+                <img className="template-img" src={ selected.image } style={ { display : variant === 'template' ? 'inline' : 'none' } }/>
+                { selected.value }
+                <span>
+                    <img src="/assets/icons/n_dropdown.svg" style={ { transform : open ? 'rotateZ(180deg)' : 'rotateZ(0deg)' } } />
+                </span>
+            </StyledSelect>
+            <StyleList style={ { display : open ? 'block' : 'none' } }>
```

Comment:
Pass the `open` as props down to `StyledList` component and handle the toggle there
```suggestion
            <StyleList isOpen={open}>
```

Reply:
TBD on Call!